src/es_sfgtools/processing/assets/tiledb_temp.py

```diff
@@ -0,0 +1,188 @@
+import tiledb
+import numpy as np
+from pathlib import Path
+from datetime import datetime
+import pandas as pd
+from pandera import check_types
+from pandera.typing import DataFrame
+
+from typing import Optional,Dict
+
+from .observables import AcousticDataFrame,GNSSDataFrame,PositionDataFrame,ShotDataFrame
+
+filters = tiledb.FilterList([tiledb.ZstdFilter(5)])
+TimeDomain = tiledb.Dim(name="time", domain=(0,), tile=100, dtype="datetime64['ms']")
+attribute_dict: Dict[str,tiledb.Attr] = {
```

Comment:
you should also compress the attributes not just the dimensions.  something like tiledb.FilterList([tiledb.ZstdFilter(level=7)]) is probably a good starting point for float32 attributes